visual-python/src/saucelabs_visual/client.py

```diff
@@ -0,0 +1,185 @@
+from os import environ
+from typing import List
+
+from gql import Client, gql
+from gql.transport.aiohttp import AIOHTTPTransport, BasicAuth
+
+from saucelabs_visual.regions import Region
+from saucelabs_visual.typing import IgnoreRegion, FullPageConfig
+
+
+class SauceLabsVisual:
+    client: Client = None
+    build_id: str = None
+
+    def __init__(self):
+        self._create_client()
+
+    def _create_client(self):
+        region_url = Region.from_name(environ.get("SAUCE_REGION") or 'us-west-1').graphql_endpoint
```

Comment:
small nit: 
if username or access key is not defined, there is no reason to read and create region URL. This code can be executed after the if statement from L23